src/engraving/style/style.cpp

```diff
@@ -422,6 +428,12 @@ void MStyle::read(XmlReader& e, compat::ReadChordListHook* readChordListHook)
             set(Sid::measureNumberPosAbove, PointF(e.readPoint()));
         } else if (tag == "measureNumberPosAbove" && m_version <= 440) { // pre-4.4 typo
             set(Sid::mmRestRangePosAbove, PointF(e.readPoint()));
+        } else if (tag == "ottavaTextAlign") {
+            // Pre-x.x scores used identical style values for Above and Below
+            // When opening older scores, use the same values for both.
+            String ottavaTextAlign = e.readText();
+            set(Sid::ottavaTextAlignAbove, TConv::fromXml(ottavaTextAlign, Align()));
+            set(Sid::ottavaTextAlignBelow, TConv::fromXml(ottavaTextAlign, Align()));
```

Comment:
Breaks a vtest:
![image](https://github.com/musescore/MuseScore/assets/1786669/bdc05b87-6ecb-43cd-909c-a715deec2393)
so let's irnore it